Shaders/DisplayDepth.fx

```diff
@@ -8,251 +8,426 @@
-		    "\n"
-			LOGARITHMIC_HELP_TEXT;
-	>;
-#else // "ui_text" was introduced in ReShade 4.5, so cannot show instructions in older versions
-	uniform bool bUIUseLivePreview <
-		ui_label = "Show live preview";
-		ui_tooltip = "Enable this to show use the preview settings below rather than the saved preprocessor settings.";
-	> = true;
-	
-	uniform int iUIUpsideDown <
-		ui_type = "combo";
-		ui_label = "Upside Down (Preview)";
-		ui_items = "RESHADE_DEPTH_INPUT_IS_UPSIDE_DOWN=0\0"
-		           "RESHADE_DEPTH_INPUT_IS_UPSIDE_DOWN=1\0";
-	> = RESHADE_DEPTH_INPUT_IS_UPSIDE_DOWN;
-	
-	uniform int iUIReversed <
-		ui_type = "combo";
-		ui_label = "Reversed (Preview)";
-		ui_items = "RESHADE_DEPTH_INPUT_IS_REVERSED=0\0"
-		           "RESHADE_DEPTH_INPUT_IS_REVERSED=1\0";
-	> = RESHADE_DEPTH_INPUT_IS_REVERSED;
-	
-	uniform int iUILogarithmic <
-		ui_type = "combo";
-		ui_label = "Logarithmic (Preview)";
-		ui_items = "RESHADE_DEPTH_INPUT_IS_LOGARITHMIC=0\0"
-		           "RESHADE_DEPTH_INPUT_IS_LOGARITHMIC=1\0";
-		ui_tooltip = "Change this setting if the displayed surface normals have stripes in them.";
-	> = RESHADE_DEPTH_INPUT_IS_LOGARITHMIC;
+uniform bool bUIShowOffset <
+    ui_label = "Blend Depth map into the image (to help with finding the right offset)";
+    ui_label_ja_jp = "透かし比較";
+    ui_tooltip_ja_jp = "補正作業を支援するために、画面効果を半透過で適用します。";
+> = false;
+
+uniform bool bUIUseLivePreview <
+    ui_category = "Preview settings";
+    ui_category_ja_jp = "基本的な補正";
+#if !ADDON_ADJUST_DEPTH
+    ui_category_toggle = true;
 #endif
+    ui_label = "Ignore preprocessor definitions (Live preview)";
+    ui_label_ja_jp = "プリプロセッサの定義を無視 (補正プレビューをオン)";
+    ui_tooltip = "Enable this to preview with the current preset settings instead of the global preprocessor settings.";
+    ui_tooltip_ja_jp =
+        "共通設定に保存されたプリプロセッサの定義ではなく、これより下のプレビュー設定を使用するには、これを有効にします。\n"
+#if ADDON_ADJUST_DEPTH
+        "設定の準備が出来たら、'設定に保存して反映する'ボタンをクリックしてから、このチェックボックスをオフにして下さい。\n\n"
+#else
+        "設定の準備が出来たら、上の'プリプロセッサの定義を編集'ボタンをクリックした後に開くダイアログに入力して下さい。\n\n"
+#endif
+        "プレビューをオンにした場合と比較して画面効果がまったく同じになれば、正しく設定が反映されています。";
+> = false;
 
-// -- Advanced options --
-#if __RESHADE__ >= 40500
-uniform int Advanced_help <
-	ui_category = "Advanced settings"; 
-	ui_category_closed = true;
-	ui_type = "radio"; ui_label = " ";
-	ui_text =
-		"\nThe following settings also need to be set using \"Edit global preprocessor definitions\" above in order to take effect.\n"
-		"You can preview how they will affect the Depth map using the controls below.\n\n"
-		"It is rarely necessary to change these though, as their defaults fit almost all games.";
-	>;
-
-	uniform bool bUIUseLivePreview <
-		ui_category = "Advanced settings";
-		ui_label = "Show live preview";
-		ui_tooltip = "Enable this to show use the preview settings below rather than the saved preprocessor settings.";
-	> = true;
+#if __RESHADE__ <= 50902
+uniform int iUIUpsideDown <
+#else
+uniform bool iUIUpsideDown <
 #endif
```

Comment:
I think this one would benefit from a `ui_text` with a simple description as well (also to separate it better from the `bUIUseLivePreview` checkbox. But I can come up with something later, not important for now.